livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py

```diff
@@ -49,10 +49,11 @@ def __init__(
         self,
         *,
         model: str | ChatModels = "gpt-4o",
+        base_url: str | None = None,
         client: openai.AsyncClient | None = None,
     ) -> None:
         self._opts = LLMOptions(model=model)
-        self._client = client or openai.AsyncClient()
+        self._client = client or openai.AsyncClient(base_url=base_url)
```

Comment:
get_base_url here too for the env var?

Reply:
AsyncClient handles it internally

Reply:
though probably good to make it consistent in case they change behavior